packages/tunnel-client/src/lib/tunnel-multiplexing-pooling-cluster.ts

```diff
@@ -0,0 +1,161 @@
+import EventEmitter from "events";
```

Comment:
This is moved from `packages/tunnel-client/src/lib/tunnel-multiplexing-cluster.ts` with no changes